plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -130,8 +130,8 @@ instance Pretty ParserError where
         "Expected a type of kind star (to later parse a constant), but got:" <+>
             squotes (pretty ty) <+> "at" <+> pretty loc
     pretty (UnknownBuiltinFunction s loc lBuiltin)   =
-        "Unknown built-in function" <+> squotes (pretty s) <+> "at" <+> pretty loc <+>
-            ". Parsable functions are " <+> pretty lBuiltin
+        "Unknown built-in function" <+> squotes (pretty s) <+> "at" <+> pretty loc <>
+            "." <> hardline <> "Parsable functions are " <+> pretty lBuiltin
```

Comment:
I changed that because the error message you currently get for an unknown builtin currently looks like this:

  |
2 | (builtin unknown)
  |                 ^
Unknown built-in function 'unknown' at "bad.uplc":2:17 . Parsable functions are  [ addInteger
                                                                                 , appendByteString
                                                                                 , appendString
                                                                                 , bData
                                                                                 , blake2b_256
                                                                                 , chooseData
                                                                                 , chooseList
                                                                                 , chooseUnit
                                                                                 , consByteString
                                                                                 , constrData
                                                                                 , decodeUtf8
                                                                                 , divideInteger
                                                                                 , encodeUtf8
                                                                                 , equalsByteString
                                                                                 , equalsData
                                                                                 , equalsInteger
                                                                                 , equalsString
                                                                                 , fstPair
                                                                                 , headList
                                                                                 , iData
                                                                                 , ifThenElse
                                                                                 , indexByteString
                                                                                 , lengthOfByteString
                                                                                 , lessThanByteString
                                                                                 , lessThanEqualsByteString
                                                                                 , lessThanEqualsInteger
                                                                                 , lessThanInteger
                                                                                 , listData
                                                                                 , mapData
                                                                                 , mkCons
                                                                                 , mkNilData
                                                                                 , mkNilPairData
                                                                                 , mkPairData
                                                                                 , modInteger
                                                                                 , multiplyInteger
                                                                                 , nullList
                                                                                 , quotientInteger
                                                                                 , remainderInteger
                                                                                 , serialiseData
                                                                                 , sha2_256
                                                                                 , sha3_256
                                                                                 , sliceByteString
                                                                                 , sndPair
                                                                                 , subtractInteger
                                                                                 , tailList
                                                                                 , trace
                                                                                 , unBData
                                                                                 , unConstrData
                                                                                 , unIData
                                                                                 , unListData
                                                                                 , unMapData
                                                                                 , verifyEcdsaSecp256k1Signature
                                                                                 , verifyEd25519Signature
                                                                                 , verifySchnorrSecp256k1Signature ]

```

With the hardline it looks like 
```
Unknown built-in function 'unknown' at "bad.uplc":2:17 . 
Parsable functions are  [ addInteger
                        , appendByteString
                        , appendString
                        , bData
                        , blake2b_256
                        , chooseData
                        , chooseList
                        , chooseUnit
                        , consByteString
                        , constrData
                        , decodeUtf8
                        , divideInteger
                        , encodeUtf8
                        , equalsByteString
                        <etc>
```
which seemed better.  I just used `hardline` because that was what was used elsewhere in the file.  Is `line` better in some way?